architecture/topics/aws_getting_started_secrets.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-secrets-{context}']
+= Using secrets to hide parameters from Service Catalog users
+
+When installing the AWS Service Broker you need to supply sensitive information such as access keys, which you can hide from Service Catalog users with {product-title} _secret_ objects. Information contained in a secret is deployed when needed, but is not displayed during the launching process.  
+
+.Procedure
+
+To create a secret to set the AWS access and secret keys and associate the secret with the AWS Service Broker:
+
+. Create a YAML file with the secret information:
++
+[source,bash]
+----
+...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-broker-keys
+stringData:
+  aws_access_key: "accesskey" <1>
+  aws_secret_key: "secretkey" <2>
+----
++
+<1> Specify the AWS Access Key for the account associated with the AWS Service Broker.
+<2> Specify the AWS Secret Access Key for the account associated with the AWS Service Broker.
++
+[NOTE]
+====  
+The `aws_access_key` and `aws_secret_key` values must match the AWS keys exactly.  If the values do not match, the parameter values will NOT receive the secret.
+====
+
+. Create the secret in the `aws-service-broker` project:
++
+[source,bash]
+----
+oc create -f aws-broker-keys.yml -n  aws-service-broker
+----
+
+. Configure the AWS Service Broker to use the secret by editing edit the configuration map for the AWS Service Broker:
++
+[source,bash]
+----
+oc edit configmap -n aws-service-broker
+----
++
+
+. Add a section for `secrets` using the following syntax:
++
+[source,bash]
+----
+    secrets:
+      - {apb_name: dh-myAPB, secret: aws-secret, title: aws-secret}
+----
++
+The `apb_name` is the name of the link:https://hub.docker.com/u/awsservicebroker[Ansible Playbook] to use with the secret.
++
+The `secret` and `title` is the name of the secret.
++
+For example:
++
+[source,bash]
+----
```

Comment:
This section looks old and references automation-broker specifics no longer valid